scripts/tests/cirque_tests.sh

```diff
@@ -38,7 +38,6 @@ CIRQUE_TESTS=(
     "EchoTest"
     "EchoOverTcpTest"
     "MobileDeviceTest"
-    "InteractionModelTest"
```

Comment:
I prefer keep this testing, currently Rob and I still use this test to generate events to validate the event generation stability.

Reply:
Looking through those tests, it doesn't look like it actually validates eventing in any meaningful way? I don't see any event parsing happening on the initiator, and I don't see any actual event emission happening on the server (I could be wrong..).

More importantly, we have more robust eventing tests that exist already in the MobileDeviceTest suite (which in turn, contains a robust IM test suite) that actually drives eventing using commands to force the emission of events, and ensuring they get received on the client.

Reply:
Per offline chat, we would try to have unit test and inject a large mount of event for this issue reproducing and root cause. At same time, we would stick with IM cirque test for issue debugging in case unit test cannot reproduce.